hashicorp/dev-portal
# dark theme: update background colors

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1203652232454021/1204107070515701) 🎟️

## 🗒️ What

Updates instances of `background-color: white` with `background-color: var(--token-color-surface-primary)` ([which is #fff](https://helios.hashicorp.design/foundations/colors?tab=palette#surface-1) in light mode) 

Also makes an override for the card component background color. 

### Affected Components:
- Card
- MobileMenuContainer
- SidebarNavMenuItem
- TabButtonControls
- TabDropdownControls
- VersionContextSwitcher dropdown
- VideoEmbed
- Integrations VariableGroupList item
- Dialog


## 🤷 Why



## 🛠️ How



## 📸 Design Screenshots



## 🧪 Testing

- [Visit the preview](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/), choose dark mode

To test the following components, switch between light and dark. Light mode should match production.
- Visit a [sidebar sidecar layout page,](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/vault) view the cards, sidebar, main content area background, hit the 'bookmark' icon to see the dialog with the dark background
- [Checkout install view](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/vault/downloads), look at the tab button controls, version switcher dropdown
- Go to [integrations component view page](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/waypoint/integrations/hashicorp/aws-ec2/latest/components/platform), checkout the group list item background
- Visit a[ tutorial with a video embed](https://dev-portal-git-ksupdate-background-color-hashicorp.vercel.app/waypoint/tutorials/get-started-docker/get-started-intro)

## 💭 Anything else?

This isn't the final state necessarily, but a first step to move away from manual color definitions and fix-up fully _broken_ background colors.
